src/helper/utils.js

```diff
@@ -0,0 +1,190 @@
+goog.module('datalayerhelper.helper.utils');
+
+const {type, hasOwn, isPlainObject} = goog.require('datalayerhelper.plain');
+
+/**
+ * Applies the given method to the value in the dataLayer with the given key.
+ * If the method is a valid function of the value, the method will be applies
+ * with any arguments passed in.
+ *
+ * @param {!Array<*>} command The array containing the key with the
+ *     method to execute and optional arguments for the method.
+ * @param {!Object<*>} model The current dataLayer model.
+ * @package
+ */
+function processCommand(command, model) {
```

Comment:
I thought we planned on moving this back into DataLayerHelper as a private (possibly static) method?

Reply:
OK! I moved it into the DataLayerHelper as a private method and changed the tests to use it indirectly.